### PR TITLE
Fixture synchronization with Migrations

### PR DESCRIPTION
**Pull request by georgeconstantinou**
Synchronize fixture fields with migration columns to keep table schema consistent.

### CODE REVIEW

File: tests/Fixture/DatabaseLogsFixture.php [hunk 1]
```diff
@@ -26,13 +26,13 @@ class DatabaseLogsFixture extends TestFixture {
 		'type' => ['type' => 'string', 'null' => false, 'default' => null, 'length' => 50],
 		'message' => ['type' => 'text', 'null' => false, 'default' => null],
 		'context' => ['type' => 'text', 'null' => true, 'default' => null],
-		'created' => ['type' => 'datetime', 'null' => true, 'default' => null],
+		'created' => ['type' => 'timestamp', 'null' => true, 'default' => null],
```
Comment (dereuromark) [inline tests/Fixture/DatabaseLogsFixture.php]: This should stay datetime afaik.

Comment (dereuromark) [reply tests/Fixture/DatabaseLogsFixture.php]: https://github.com/dereuromark/CakePHP-DatabaseLog/blob/master/config/Migrations/20161005230232_InitDatabaseLogs.php#L36
is maybe the wrong one?

Comment (dereuromark) [reply tests/Fixture/DatabaseLogsFixture.php]: Given the fact that the current code uses timestamp I would merge this now. We should probably fix the migration in general in a feature minor release.

Comment (georgeconstantinou) [reply tests/Fixture/DatabaseLogsFixture.php]: Thank you @dereuromark

**Pull request merged**